Adafruit_Video_Looper/video_looper.py

```diff
@@ -126,8 +128,50 @@ def _is_number(iself, s):
             return True
         except ValueError:
             return False
-    
+
     def _build_playlist(self):
```

Comment:
there is a big issue in this function is just realised.
if player is in usb_mode and an playlist path is set and no usb drive is connected the player will just exit with a runtime error since playlist path does not exist... this is obvious not ideal...
I created a pull request against your branch for a fix (and some other tweaks)
https://github.com/KoltesDigital/pi_video_looper/pull/1